redis/mutation_utils.hh

```diff
@@ -34,7 +34,7 @@ namespace redis {
 class redis_options;
 
 future<> write_hashes(service::storage_proxy& proxy, redis::redis_options& options, bytes&& key, bytes&& field, bytes&& data, long ttl, service_permit permit);
-future<> write_strings(service::storage_proxy& proxy, redis::redis_options& options, bytes&& key, bytes&& data, long ttl, service_permit permit);
+future<bool> write_strings(service::storage_proxy& proxy, redis::redis_options& options, bytes&& key, bytes&& data, long ttl, service_permit permit, smp_service_group& ssg);
```

Comment:
As I noted above, cas() returns true or false because it allows apply() to decide - considering the previous item's value (qr) - that some condition isn't met, and the operation should be aborted. If there is no condition - as in your "write_strings", it doesn't make sense to return a bool. This operation will always succeed.
(Of course, the operation can also fail on timeout, i/o error, etc., but this will be an exception as usual - it's not this bool).

Reply:
Reverted the code that `write_strings()` to return `future<bool>`.